gui-apps/river-tag-overlay/river-tag-overlay-1.0.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3
+
+DESCRIPTION="A pop-up showing tag status [in the river window manager]"
+HOMEPAGE="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+#SRC_URI="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+EGIT_REPO_URI="https://git.sr.ht/~leon_plickat/river-tag-overlay"
+
+LICENSE="GPLv3"
+SLOT="0"
+KEYWORDS="~arm64 ~amd64"
+
+BDEPEND="sys-devel/make dev-util/pkgconf"
```

Comment:
put them in separate lines.
Don't need to add make here.
Use virtual/pkgconfig instead of dev-util/pkgconf - https://packages.gentoo.org/packages/virtual/pkgconfig
You also need wayland-scanner in BDEPEND.